ext/ddtrace.h

```diff
@@ -147,6 +147,7 @@ ZEND_BEGIN_MODULE_GLOBALS(ddtrace)
     HashTable telemetry_spans_created_per_integration;
     ddog_SidecarActionsBuffer *telemetry_buffer;
 
+    uint8_t trace_source_bm;
```

Comment:
This should not be maintained on the globals, but it's part of specific traces. Each trace can have its indvidual trace source.

Reply:
I didn't add it to the root span until the last possible moment. The reason is that on the rootSpan it's a string and for performance purposes, I want to keep it as integer to avoid doing conversions back and forth. Does it makes sense? Not sure if you meant that

Reply:
The problem is, that multiple traces may exist in parallel. However if _any_ trace has an asm event, it gets set on the trace and no further trace (within the same RINIT-lifecycle) will be applied the asm trace source.